Form/Type/CollectionsType.php

```diff
@@ -15,32 +17,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add(
                 'languages',
-                'collection',
```

Comment:
This is deprecated and will be removed in 3.0, see https://github.com/symfony/symfony/blob/master/UPGRADE-3.0.md#form